### PR TITLE
Supported a fake click in Dart Debug Extension for testing.

### PR DESCRIPTION
**Pull request by pisimulation**
- WebDriver does not support clicking on the extension icon. Added a fake click listener to trigger the extension for E2E testing.

### CODE REVIEW

File: dwds/debug_extension/web/background.dart [hunk 1]
```diff
@@ -55,7 +55,13 @@ void main() {
     queryTabs(query, allowInterop((List tabs) {
       callback(List.from(tabs));
     }));
-  }));
+  });
+  addListener(startDebug);
+
+  // For testing only.
+  onFakeClicked = allowInterop(() {
```
Comment (grouma) [inline dwds/debug_extension/web/background.dart]: [nit] onFakeClick

File: dwds/debug_extension/web/background.dart [hunk 2]
```diff
@@ -55,7 +55,13 @@ void main() {
     queryTabs(query, allowInterop((List tabs) {
       callback(List.from(tabs));
     }));
-  }));
+  });
+  addListener(startDebug);
+
+  // For testing only.
+  onFakeClick = allowInterop(() {
+    startDebug(null);
```
Comment (natebosch) [inline dwds/debug_extension/web/background.dart]: we unconditionally reference `e.result.value` - is `null` a valid argument here?

Comment (pisimulation) [reply dwds/debug_extension/web/background.dart]: the `e` in `e.result.value` is from line 39. The `e` on line 23 is never used. What's the best way to name an unused but required parameter?

Comment (natebosch) [reply dwds/debug_extension/web/background.dart]: ah cool, the idiomatic name for an unused argument is `_`

**Pull request merged**